src/index.js

```diff
@@ -7,7 +7,7 @@ import {
 export function isFSA(action) {
   return (
     isPlainObject(action) &&
-    (isString(action.type) || isSymbol(action.type)) &&
```

Comment:
There is an import that needs to be removed as well.

Reply:
Thanks. I was editing on github directly. Missed the import. 😛